owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/renderer/service/RendererServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package fr.openwide.core.showcase.web.application.renderer.service;
+
+import fr.openwide.core.showcase.web.application.ShowcaseApplication;
+import fr.openwide.core.wicket.more.rendering.service.AbstractRendererServiceImpl;
+
+public class RendererServiceImpl extends AbstractRendererServiceImpl {
+	
+	@Override
+	protected String getApplicationName() {
```

Comment:
I agree the current behavior is not that nice. Maybe it could come as another PR?